infra/update_zuul_layout/update-zuul-layout.py

```diff
@@ -5,6 +5,7 @@
 import sys
 import re
 
+# TODO: This script assumes that it's being ran from inside ansible/ directory.
 layout_file = "roles/zuul/files/layout.yaml"
```

Comment:
It seems that value of layout_file is the only thing that should be updated. We can probably avoid some problems (and questions) if we change it together with moving the script.